functionary/requirements.in

```diff
@@ -10,6 +10,7 @@ psycopg2
 pydantic
 pyyaml
 redis
+thefuzz
```

Comment:
This complains about speed without the python-Levenshtein package also being installed.

Reply:
The python-Levenshtein library doesn't seem to be actively maintained, so I choose to not include it

Reply:
Silenced the warning

Reply:
python-Levenshtein is a wrapper around levenshtein which has been active this year, last commit in October. Installing levenshtein also installs rapidfuzz which looked like it might be a drop-in for this usecase.

Reply:
I'll check out rapidfuzz. That project seems much more active than thefuzz.

thefuzz repo references [this](https://github.com/ztane/python-Levenshtein/) project as the Levenshtein dependency, which is why I didn't install it

Reply:
Switched libraries to RapidFuzz